deepcell_tracking/utils.py

```diff
@@ -203,23 +206,86 @@ def save_trks(filename, lineages, raw, tracked):
     """Saves raw, tracked, and lineage data into one trks_file.
 
     Args:
-        filename (str): full path to the final trk files.
-        lineages (dict): a list of dictionaries saved as a json.
+        filename (str or io.BytesIO): full path to the final trk files or bytes object
+            to save the data to
+        lineages (list): a list of dictionaries saved as a json.
         raw (np.array): raw images data.
         tracked (np.array): annotated image data.
 
+    Returns:
+        io.BytesIO: If filename is instance of io.BytesIO
+
     Raises:
         ValueError: filename does not end in ".trks".
     """
-    if not str(filename).lower().endswith('.trks'):
+    if not str(filename).lower().endswith('.trks') and not isinstance(filename, io.BytesIO):
```

Comment:
this line is so long! Can we break this up into two conditionals? or save one of them as a variable? 

Please also fix up the other check in save_track_data.